arc-script/arc-script-core/src/compiler/mlir/display.rs

```diff
@@ -414,7 +414,7 @@ pretty! {
             mlir::ScalarKind::DateTime => write!(w, "ui64"),
             mlir::ScalarKind::Duration => write!(w, "ui64"),
             mlir::ScalarKind::Str      => crate::todo!(),
-            mlir::ScalarKind::Char     => crate::todo!(),
+            mlir::ScalarKind::Char     => write!(w, "char"),
```

Comment:
Should this be in this commit? You change it in a later commit.

Reply:
Oops, the first commit should also probably come in later